org-formation/075-security-hub/security-hub-suppress-infra.yaml

```diff
@@ -382,3 +382,39 @@ Resources:
           - SecurityHubFindingsQueue
           - Arn
         Id: Target0
+
+  # This rule suppresses findings with the given generator IDs only for the bootstrap account and service accounts
+  SuppressFindingsForPublicBucketsRule:
```

Comment:
this name doesn't seem to match what's being supressed?

Reply:
Fixed